packages/firestore/lib/index.d.ts

```diff
@@ -175,6 +175,24 @@ export namespace FirebaseFirestoreTypes {
    */
   export type DocumentChangeType = 'added' | 'removed' | 'modified';
 
+  /**
+   * The types for a DocumentSnapshot field that are supported by Firestore.
+   */
+  export type DocumentFieldType =
+    | string
```

Comment:
Apologies. A couple of other things that I missed. the `|` operator shouldn't be on line 182.

Reply:
This operator is indeed redundant, but harmless. It's automatically generated by Prettier for consistent styling.